### PR TITLE
I have improved Readme.md for art-marketplace project

### PR DESCRIPTION
**Pull request by heliomar-pena**
You can check the new readme in this link: https://github.com/Azordev/art-marketplace/blob/95bb459bb61860b37d3bdffe425e78c99295312d/README.md

imagen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,30 +17,31 @@
...
+  - [Collaborators](#collaborators)
   - [Contributing](#contributing)
   - [Show your support](#show-your-support)
   - [License](#license)
 
-## The Project
+## Art Marketplace Docs
 
-A place where you can find a download BIM objects
+A place where you can find a download Art images
```
Comment (Israel-Laguan) [inline README.md]: Maybe mention here that we use the endpoint for real art

Comment (Israel-Laguan) [reply README.md]: Can I ask for some kind of icon or image @angelik0828? To use as a favicon and to put in this document

Comment (heliomar-pena) [reply README.md]: What do you think now? If u think that can improve the text let me know

**Comment by heliomar-pena** [comment]
I have deleted for mistake the issue and pull request templates, I should solve it before merge

**Review by heliomar-pena — COMMENTED** [review]
Rebase done, please check

**Review by Israel-Laguan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Israel-Laguan) [reply README.md]: @Katsu08 we can add this image later in other issue

**Pull request merged**